lib/omnibus/templating.rb

```diff
@@ -61,10 +55,22 @@ def render_template(source, options = {})
           Struct.new(*variables.keys).new(*variables.values)
         end
 
-      result = template.result(struct.instance_eval { binding })
+      @result = template.result(struct.instance_eval { binding })
+      @result
```

Comment:
This method should just return the result of `template.result` directly instead of setting that value to an instance variable (`@result`) and turning around and returning the instance variable.